style.css

```diff
@@ -0,0 +1,3 @@
+.header-style{
+    color: green;
+}
```

Comment:
- [ ] Kindly fix the linters which are not passing for the css, you can find more information on how to solve the linters by clicking on details and it would show you what linters aren't passing so you can fix them accordingly. :+1: